SSSD/sssd
# Tests: Add passkey test cases for following scenario

Test cases are as follows:
7.  Check offline authentication of a user with LDAP, IPA, AD and Samba
8.  Fetch user from cache for LDAP, IPA, AD and Samba server
9.  Check authentication of user when multiple keys added for same user with LDAP, IPA, AD and Samba server.
10. Check authentication of user when same key added for multiple user with LDAP, IPA, AD and Samba server.
